tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/TensorFlow.scala

```diff
@@ -48,7 +39,7 @@ trait TensorFlow extends py.Object {
 
   def zeros(shape: Seq[Int]): Tensor = py.native
 
-  def reshape(tensor: Tensor, shape: PythonList[Int]): Tensor = py.native
+  def reshape(tensor: Tensor, shape: Tensor): Tensor = py.native
```

Comment:
Hmm shape can be a tensor itself? :hushed: so... what's the reshape logic in case `shape` is a 2+ dimensional tensor (and not just "traditionally" a 1-dimensional one)?

Reply:
oh no no. shape is a tensor but only 1D - logically. 
If you are confused look here: https://www.tensorflow.org/api_docs/python/tf/reshape#args

Reply:
Ok, but still - change of type from sth that's purely 1D (`PythonList`) to sth that's possibly 2D+ (`Tensor`) reduces type safety... although I do see that the underlying Python expects a `Tensor`.
Hmm how we maintain both 1) type safety and 2) interoperability with Python and just pass a `Seq[Int]` to this Scala method, only to convert it to a 1D `Tensor` before passing to the underlying Python?

Reply:
sure sounds good